### DISCUSSION TITLE
Dumped games are locked/not playable

### DISCUSSION THREAD

**Discussion by TheDcoder** [hippie68/ftpdump#18] [Q&A] [answered]
Hi,

I have used your excellent script and FTP daemon to dump some games but they are not playable, the dumped games have a lock icon and if I try to launch them (with GoldHEN enabled) I get an error that I should purchase the content before I can play it, this is the same error which occurs when you install a retail `pkg` on a jail-broken system.

So far I have dumped two games (GTA V and LittleBigPlanet 3) and all of them have the same issue, I am not sure if I am doing anything wrong, here are the command I ran to dump the games:
```
ps4-ftpdump --app --patch --dump CUSA00063 192.168.1.206
```

The dumping process is successful without any errors and the packages install without any errors in the PS4 too, however I just can't get the games to work.

I am running 9.00 firmware, I hope that is okay?

**Comment by hippie68** [comment, marked as answer]
I realize that option's name may be misleading: --dump TITLE_ID downloads existing PKG files. What you may want to do instead is launch a game and enter:

    ps4-ftpdump 192.168.1.206

or, if you don't want to dump DLC:

    ps4-ftpdump --app --patch 192.168.1.206

Please note that, like current dumper payloads, the script does not create PKG files.

**Reply by TheDcoder** [reply]
Ah, I see! It would be nice if you could modify the help text to make it extra clear that `--dump TITLE_ID` only dumps existing retail PKG files.

> Please note that, like current dumper payloads, the script does not create PKG files.

Got it, can you recommend a tool which can be used to create a PKG file from the dumped contents? Maybe you can make your own program and integrate it into this script? :)
